_docs/tasks/rate-limiting.md

```diff
@@ -83,118 +137,51 @@ Using Istio we can ensure that `1qps` is not breached.
 ## Conditional rate limits
 
 In the previous example we applied a rate limit to the `ratings` service without regard
-to any other attributes. It is possible to conditionally apply rate limits based on 
-attributes like the source of the traffic.
-
-The following configuration applies a `1qps` rate limit only to version `v3` of `reviews`.
-
-1. Configure mixer with the conditional rate limit.
-
-   Save this as ratelimit-conditional.yaml:
+to non dimension attributes. It is possible to conditionally apply rate limits based on arbitrary
+attributes using a match condition in the quota rule.
 
    ```yaml
-   rules:
-   - selector: source.labels["app"]=="reviews" && source.labels["version"] == "v3"  
-     aspects:
-     - kind: quotas
-       params:
-         quotas:
-         - descriptorName: RequestCount
-           maxAmount: 1
-           expiration: 1s
-   ```
+   apiVersion: config.istio.io/v1alpha2
+   kind: rule
+   metadata:
+     name: quota
+     namespace: istio-config-default
+   spec:
+     match: source.namespace != destination.namespace
+     actions:
+     - handler: handler.memquota
+       instances:
+       - requestcount.quota
 
-   and then run the following command:
-
-   ```bash
-   istioctl mixer rule create global ratings.default.svc.cluster.local -f ratelimit-conditional.yaml
    ```
-
-   Notice the rule is the same as the previous example, only this one uses a `selector` to
-   apply the ratelimit only for requests from `reviews:v3`.
-   
-1. Generate load on the `productpage` with the following command:
-
-   ```bash
-   while true; do curl -s -o /dev/null http://$GATEWAY_URL/productpage; done
-   ```
-
-1. Refresh the `productpage` in your browser.
-
-   As in the previous example, while the load 
-   generator is running (i.e., generating more than 1 req/s), the traffic generated by
-   your browser will be rate limited, but this time only if the request is from `reviews:v3`.
-   Notice that this time if you log in as user "jason" (the `reviews:v2` user) you will continue to see
-   the black ratings stars. Only the other users will stop seeing
-   the red ratings stars while the load generator is running.
+The quota rule will now only be applied if the source and the destination namespace of the request
```

Comment:
This configuration applies the quota rule to requests whose source and destination namespaces are different.